### ISSUE TITLE
[Network] NetworkWatcher tests will not re-record

### ISSUE THREAD

**Issue by tjprescott** [Azure/azure-cli#6009]
The test is flaky and intermittently fails on:
```
msrest.exceptions.ClientRequestError: Error occurred in request., RetryError: HTTPSConnectionPool(host='management.azure.com', port=443): Max retries exceeded with url: /subscriptions/0b1f6471-1bf0-4dda-aec3-cb9272f09590/resourceGroups/cli_test_network_watcher3kwfxgi6xzydmjnnozhc3hpcto3ixlnsfejv3gq4et7ob7mee7h/providers/Microsoft.Network/networkWatchers/westcentralus-watcher/ipFlowVerify?api-version=2018-02-01 (Caused by ResponseError('too many 500 error responses',))
```

The test has been converted into `LiveScenarioTest` to facilitate merging the latest Network API PR. The fix will need to be a way of separating the NetworkWatcher resource tests from the different operational tests so that we don't need to essentially disable the entire NetworkWatcher test because one piece is being flaky.

**Comment by adewaleo** [comment]
Was this fixed in #7165 @tjprescott ?

**Comment by tjprescott** [comment]
Yes.
